### DISCUSSION TITLE
Bring up the init command in readme

### DISCUSSION THREAD

**Discussion by AmazingTurtle** [helmfile/helmfile#661] [Questions] [answered]
I thought I knew helmfile when I skimmed the docs. After hours of researching I figured that I was missing the `diff` plugin. Upon further review I found that there is a `helmfile init` command that takes care of that installation. I wish I read about that earlier. The [Installation](https://github.com/helmfile/helmfile/#installation) process is so streamlined, that I thought that'd be it.

I suggest to add a hint to run `helmfile init` once after installing to make sure that it is _correctly_ installed.

**Comment by yxxhero** [comment, marked as answer]
@AmazingTurtle PR is welcome. Thanks very much.

**Reply by AmazingTurtle** [reply]
https://github.com/helmfile/helmfile/pull/662
